acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/web3j/RevertReason.sol

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2018 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+pragma solidity >=0.4.0 <0.6.0;
+
+// compile with:
+// solc RevertReason.sol --bin --abi --optimize --overwrite -o .
+// then create web3j wrappers with:
+// web3j solidity generate -b ./generated/RevertReason.bin -a ./generated/RevertReason.abi -o ../../../../../ -p tech.pegasys.pantheon.tests.web3j.generated
+contract RevertReason {
+
+    uint public value = 0;
+
+    function revertWithRevertReason() public {
+        if (true) {
```

Comment:
Why not use the `assert` and `require` operations for these two methods instead of the redundant if/else?

Reply:
simplified the test contract to just revert